.jules/runs/steward_release/decision.md

```
@@ -0,0 +1,17 @@
+# Decision
+
+## 🧭 Options considered
+
+### Option A (recommended)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)